modules/graceful-restart.adoc

```diff
@@ -172,12 +172,12 @@ Check that the status for all nodes is `Ready`.
 [source,terminal]
 ----
 NAME                           STATUS   ROLES                  AGE   VERSION
-ip-10-0-168-251.ec2.internal   Ready    control-plane,master   82m   v1.31.3
-ip-10-0-170-223.ec2.internal   Ready    control-plane.master   82m   v1.31.3
-ip-10-0-179-95.ec2.internal    Ready    worker                 70m   v1.31.3
-ip-10-0-182-134.ec2.internal   Ready    worker                 70m   v1.31.3
-ip-10-0-211-16.ec2.internal    Ready    control-plane,master   82m   v1.31.3
-ip-10-0-250-100.ec2.internal   Ready    worker                 69m   v1.31.3
+ip-10-0-168-251.ec2.internal   Ready    control-plane,master   82m   v1.32.3
+ip-10-0-170-223.ec2.internal   Ready    control-plane.master   82m   v1.32.3
```

Comment:
Unrelated to your change, but this looks like a typo?
```suggestion
ip-10-0-170-223.ec2.internal   Ready    control-plane,master   82m   v1.32.3
```

Reply:
Could be, but that's an odd place to have a typo. I can't imagine someone was messing with that output... :thinking:

Reply:
@jeana-redhat opened #93000. Thanks for pointing it out!

Reply:
what a soothing PR number to resolve it :relieved: ty!